operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -82,15 +82,24 @@ Release dates are currently communicated in the following ways. Further iteratio
 
 Release Manager is able to look at the version we have deployed to Mattermost Cloud. Also, when we are hooked up to split.io, there will be a dashboard where you can see active and historic feature flags. Any feature that has its flag removed would be included on any Self-Managed releases past the point after that removal was merged. More details on feature flags: https://developers.mattermost.com/contribute/server/feature-flags/.
 
+## Cloud Release Branch Processes
+
+**Process for merging the ``master`` branch into the ``cloud`` branch**
```

Comment:
@ethervoid My goal is to document this process for visibility for devs who may not know how it works yet. Let me know if any steps or details should be added below.